build.gradle

```diff
@@ -115,10 +115,6 @@ repositories {
     maven {
         url "http://maven.shadowfacts.net/"
     }
-    maven {
```

Comment:
Maybe not remove it entirely and just comment it out, Player told me he'd fix it, he just can't do it right now. Not sure how long it'll take though

Reply:
I'd say if we "need" it again, we add it again. The maven cache I built has all our dependencies cached 30 days and acts as a proxy for any new artifacts.

In fact, gradle is built this way that if one repository does not answer at all, the build fails. The reason was that there are people with (imo nasty) setups that need reproducible builds. There is also no switch or config option to work around one repository timing out. That's why I built our cache and that's why I'd rather kick out any repository that fails sometimes.